File: java/URI_1046.java
package URI;
import java.util.Scanner;

public class URI_1046 {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int inicio = sc.nextInt();
		int fim = sc.nextInt();
		int duracao;
		
		if(inicio < fim) {
			duracao = fim - inicio;			
		}
		else {
			duracao = 24 - inicio + fim;
		}
		
		System.out.println("O JOGO DUROU " + duracao + " HORA(S)");
		sc.close();		

	}

}

File: kotlin/URI_1046.kt
import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)

    val horaInicial = sc.nextInt()
    val horaFinal = sc.nextInt()

    val duracao = when {
        horaInicial < horaFinal -> horaFinal-horaInicial
        else -> 24 - horaInicial + horaFinal
    }
    println("O JOGO DUROU $duracao HORA(S)")
}
